tests/conftest.py

```diff
@@ -481,3 +482,30 @@ def teardown_tftp_storage():
         assert teardown.values()[0]["rc"] == 0
 
     request.addfinalizer(teardown_tftp_storage)
+
+
+@pytest.fixture(scope="function")
+def setup_yum_content(request, ansible_module):
+    """ Setup/Teardown cusom yum repo for test_positive_content_migrate."""
+    prod_name = "Animals"
+    repo_name = "Animal_repo"
```

Comment:
I feel we should use `fauxfactory` to generate these names using `gen_string("alpha")`

Reply:
Yes, fauxfactory would be the proper way for naming.